pkg/client/remotecommand/remotecommand.go

```diff
@@ -66,26 +68,26 @@ func New(req *client.Request, config *client.Config, command []string, stdin io.
 // Execute sends a remote command execution request, upgrading the
 // connection and creating streams to represent stdin/stdout/stderr. Data is
 // copied between these streams and the supplied stdin/stdout/stderr parameters.
-func (e *RemoteCommandExecutor) Execute() error {
+func (e *Executor) Execute() error {
 	doStdin := (e.stdin != nil)
 	doStdout := (e.stdout != nil)
 	doStderr := (!e.tty && e.stderr != nil)
 
 	if doStdin {
-		e.req.Param(api.ExecStdinParam, "1")
+		e.req.Param("stdin", "true")
```

Comment:
@smarterclayton they weren't deleted, just not applicable anymore. They were for the internal endpoint. I will use the api.ExecOptions object to do the conversion.